src/vl-map.js

```diff
@@ -204,6 +211,16 @@ export class VlMap extends vlElement(HTMLElement) {
     this.map.zoomToExtent(boundingbox, max);
   }
 
+  /**
+   * Zoomt op de kaart naar geometrie.
+   *
+   * @param {Geometry} geometry
+   * @param {Number} max
+   */
+  zoomToGeometry(geometry, max) {
+    this.map.zoomToGeometry(geometry, max);
+  }
+
```

Comment:
Ik zou overwegen om `zoomTo` uit te breiden met geometry argument. Afnemers kunnen dan een boundingbox of geometry meegeven en moeten dan niet op basis van argument type een andere functie aanroepen.

Reply:
had ik ook opgemerkt in iets andere vorm denk ik

Reply:
> @coemans
> was met pascal nog eens door het zoomToGeometry verhaal gegaan en hij was van mening dat het op zich wel ok is omdat ik nu een bestaande methode uit de vl-mapactions gebruik welke al publiek beschikbaar was. We hebben wel besloten om de methode zoomTo op vl-map te hernoemen naar zoomToBoundingbox. Maar dit is wel een breaking change wat we zeker in de release moeten vermelden!

Ik zou willen voorkomen dat er binnenkort 10 zoomTo functies zijn die allemaal kanshebbers zijn voor breaking changes. Ik ben overtuigd dat de afnemer wenst te zoomen naar iets en zijn API niet wilt wijzigen afhankelijk van het argument. Ik zou nog steeds zoomTo uitbreiden zodat meerdere types van argumenten ondersteund zijn.

PS: De API van vl-mapactions is inderdaad al publiek, maar moet hier niet 1 op 1 overgenomen worden.

Reply:
Ik vind dat zoals in Java een methode maken die Object aanpakt en dan maar wa instance of doen ... dus ik zie liever methodes met duidelijke namen dan ene methode waar ge moet "raden" wat die allemaal aanpakt.

Reply:
De typed vs. non typed languages discussie zal ons te ver brengen. Laat ik je (proberen om je te) overtuigen met het feit dat OL het ook toepast:
```JS
// Fit the given geometry or extent based on the given map size and border. The size is pixel dimensions of the box to fit the extent into.
fit(geometryOrExtent, opt_options)
```

Reply:
En dan daarna om "doSomething(whatever)" te kunnen doen zeker :-)
Ja soit, ik ga hier niet voor dwarsliggen maar ik heb liever helderderderdere methodes. Maar your call.

Reply:
ik bekijk dit meteen nog even verder

Reply:
bij deze, methode aangepast zodat die zowel een boudingbox als geometry aanvaard.
Test hiervoor toegevoegd en bestaande test cases aangepast.